README

```diff
@@ -355,12 +355,26 @@ dosbox -opencaptures program
 
   -machine machinetype
         Setup DOSBox to emulate a specific type of machine. Valid choices are:
-        hercules, cga, ega, pcjr, tandy, svga_s3 (default) as well as
-        the additional SVGA chipsets listed in the DOSBox configuration file.
-        svga_s3 enables VESA emulation as well.
+        - hercules      - Hercules Graphics Card (monochrome)
+        - cga           - IBM Color Graphics Adapter
+        - cga_mono      - IBM CGA attached to monochrome display (monochrome)
+        - ega           - IBM Enhanced Graphics Adapter
+        - vgaonly       - IBM Video Graphics Array (see below)
+        - pcjr          - IBM PCjr
+        - tandy         - Tandy Graphics Adapter (Tandy 1000)
+        - vesa_oldvbe   - VESA SVGA - VESA BIOS Extensions (VBE) 1.2
+        - vesa_nolfb    - VESA SVGA - VBE 2.0 with Linear Frame Buffer disabled
+        - svga_s3       - VESA SVGA - S3 Trio - VBE 2.0
+        - svga_et3000   - VESA SVGA - Tseng ET3000 - VBE 2.0
+        - svga_et4000   - VESA SVGA - Tseng ET4000 - VBE 2.0
+        - svga_paradise - VESA SVGA - Paradise Systems PVGA1A - VBE 2.0
+
+        The default is svga_s3.
+
```

Comment:
Suggest sorting from oldest/most-limited to newest/most-versatile. 
1. hercules
2. cga_mono
3. cga
4. pcjr
5. tandy
6. ega
7. vgaonly
8. vesa_oldvbe
9. paradise
10. et3000
11. et4000
12. s3 (Trio)
13. vesa_nolfb

Reply:
(and matched ordering in doc nroff)

Reply:
- cga / cga_mono - I'll stick to existing order (they are functionally equivalent - cga_mono is DOSBox community invention)
- tandy and pcjr before ega - I agree, reordered
- vesa_nolfb as the last one - I do not agree - disabling LFB is limitation, not an enhancement. This artificial limitation fixes some games, but it's a limitation nonetheless.
- paradise/et*/s3 - I have no idea how powerful those cards were originally, but I am ok with changing order and placing s3_trio at last position

Reply:
Order looks good and matches the one here:

https://en.m.wikipedia.org/wiki/Video_Graphics_Array

Paradise lower than ET3000 in entry level class.
ET 4000 mid-range.
S3 Trio high-end.